docs/source/guides/monitoring.rst

```diff
@@ -32,13 +32,13 @@ Preface
 Local Deployment
 ----------------
 
-This section will walk you through how to setup the stack locally, with the optional guide on using ``docker-compose`` for easy deployment of the stack.
+This section will walk you through how to set up the stack locally, with the optional guide on using ``docker-compose`` for easy deployment of the stack.
```

Comment:
I don't think this is a typo.

Reply:
setup is a noun, and set up is a verb phrase.

You can check: https://www.merriam-webster.com/dictionary/setup